oci/mock-rock/_releases.json

```diff
@@ -7,18 +7,29 @@
             "target": "latest_candidate"
         },
         "edge": {
-            "target": "latest_beta"
+            "target": "1.1-22.04_edge"
         }
     },
     "1.0-22.04": {
         "candidate": {
-            "target": "47"
+            "target": "67"
         },
         "beta": {
             "target": "1.0-22.04_candidate"
         },
         "edge": {
-            "target": "47"
+            "target": "67"
+        }
+    },
+    "1.1-22.04": {
```

Comment:
this is still wrong. your test only passed cause you had a previous build that wrongfully tagged revision 68 as part of track 1.1-22.04, which cannot be. Our example must be pristine, otherwise we are not leading by example.

you'll need to manually remove this track from this Json file, and change image.yaml to not point here anymore, creating a new channel instead, that is sane and points to a valid revision/channel.

Reply:
Normally now it's good: https://github.com/canonical/oci-factory/actions/runs/5147411507

Reply:
it's still not tested with a new channel

Reply:
here: https://github.com/canonical/oci-factory/actions/runs/5155071403